src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -1718,6 +1747,13 @@ static int do_map(int argc, const char *argv[], Config *cfg, bool reconnect)
 
   use_netlink = cfg->try_netlink || reconnect;
   if (use_netlink) {
+    if (!reconnect) {
+      /* FIXME: If kernel version doesn't suppot NBD_ATTR_BACKEND_IDENTIFIER
+       * we should skip generating uuid for cookie
+       */
```

Comment:
nit: This comment does not look useful to me. As I understand we are going just to always try to set the cookie. Do you really plan to add a check if the kernel support NBD_ATTR_BACKEND_IDENTIFIER?

Reply:
At some point I thought there is a need for it, but now I don't think we need one.
I will remove the commnet.

Reply:
At some point, I thought there is a need for it, but now I don't think we need one.
I will remove the commnet.